runtime/src/asset.rs

```diff
@@ -961,18 +963,22 @@ decl_module! {
             ticker.canonize();
             ensure!(Self::is_owner(&ticker, did), "user is not authorized");
 
-            <Documents<T>>::insert((ticker, name), (uri, document_hash, <pallet_timestamp::Module<T>>::get()));
-            Ok(())
+            for doc in documents.iter() {
+                Self::add_document(ticker, doc.clone());
+            }
         }
 
-        /// Used to remove the document details for the given token, Only be called by the token owner
+        /// Update document details for the given token, Only be called by the token owner
         ///
         /// # Arguments
         /// * `origin` Signing key of the token owner
         /// * `did` DID of the token owner
         /// * `ticker` Ticker of the token
-        /// * `name` Name of the document
-        pub fn remove_document(origin, did: IdentityId, ticker: Ticker, name: Vec<u8>) -> DispatchResult {
```

Comment:
A function to remove documents is required.

Reply:
Added `remove_documents(origin, did: IdentityId, ticker: Ticker, doc_ids: Vec<u64>)` which takes a vec of doc_ids to be removed.